CombinedResourceManager.cs

```diff
@@ -103,11 +101,30 @@ public override IList<ResourceRequiredContext> BuildRequiredResources(string str
 
                 DiscoverResourceOverrides(resources, resourceType);
 
-                IList<ResourceRequiredContext> result;
+                IList<ResourceRequiredContext> result = null;
 
-                if (resourceType == ResourceType.Style) result = _combinatorService.CombineStylesheets(resources, settings);
-                else if (resourceType == ResourceType.JavaScript) result = _combinatorService.CombineScripts(resources, settings);
-                else return base.BuildRequiredResources(stringResourceType);
+                for (int retry = 0; retry < 3; retry++)
+                {
+                    try
+                    {
+                        if (resourceType == ResourceType.Style) result = _combinatorService.CombineStylesheets(resources, settings);
+                        else if (resourceType == ResourceType.JavaScript) result = _combinatorService.CombineScripts(resources, settings);
+                        else return base.BuildRequiredResources(stringResourceType);
+
+                        break;
+                    }
+                    catch (Exception ex)

```

Comment:
Also check in a `when` if this is the exception you're looking for.

```suggestion
                    catch (IOException ex)

```